src/renderer/runner.ts

```diff
@@ -281,7 +280,7 @@ export class Runner {
     }
 
     const name = await this.appState.getName();
-    const appData = getAppDataDir(name);
+    const appData = this.appState.getAppDataDir(name);
```

Comment:
Feel like we could get rid of `this.appState.getAppDataDir()` and just call `path.join` directly here.

Reply:
171b2ae